### PR TITLE
filter

### PR DESCRIPTION
**Pull request by chuanchyone1**
git push -u origin filter

### CODE REVIEW

File: Bolt T-Shirt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+  aaa[m
```
Comment (JuanDelahozMiranda) [inline Bolt T-Shirt]: Seems this file you don't need it

**Review by JuanDelahozMiranda — APPROVED** [review]

File: Bolt T-Shirt (hunk 1 above)
Comment (chuanchyone1) [reply Bolt T-Shirt]: change made

File: cypress/page/products-list.page.ts [hunk 2]
```diff
@@ -2,11 +2,11 @@ class ProductListPage {
...
 
     constructor() {
-        this.sauceLabsBolt = "#item_1_title_link";
+        this.sauceLabsBolt = ".inventory_list .inventory_item";
     }
 
-    public selectSauceLabsBolt(): void {
-        cy.get(this.sauceLabsBolt).click();
+    public findProductByName(): void {
+        cy.get(this.sauceLabsBolt).filter(':contains("Sauce Labs Bolt T-Shirt")').click();
```
Comment (danielgalvis98) [inline cypress/page/products-list.page.ts]: You can receive the product name by paramenter

Comment (chuanchyone1) [reply cypress/page/products-list.page.ts]: change made

image

File: cypress/page/products-list.page.ts [hunk 3]
```diff
@@ -2,11 +2,11 @@ class ProductListPage {
...
 
     constructor() {
-        this.sauceLabsBolt = "#item_1_title_link";
+        this.sauceLabsBolt = ".inventory_list .inventory_item";
     }
 
-    public selectSauceLabsBolt(): void {
-        cy.get(this.sauceLabsBolt).click();
+    public findProductByName(nombre_producto: string): void {
+        cy.get(this.sauceLabsBolt).filter(':contains(' + nombre_producto + ')').click();
```
Comment (danielgalvis98) [inline cypress/page/products-list.page.ts]:
```suggestion
        cy.get(this.sauceLabsBolt).filter(`:contains(${nombre_producto})`).click();    
```

Comment (chuanchyone1) [reply cypress/page/products-list.page.ts]: change made

image

**Review by danielgalvis98 — APPROVED** [review]

**Pull request merged**